README.md

```diff
@@ -12,439 +12,91 @@
     <img src="https://img.shields.io/github/issues-pr/lstein/stable-diffusion?logo=GitHub&style=for-the-badge" alt="pull-requests"/>
 </p>
 
-This is a fork of CompVis/stable-diffusion, the wonderful open source
-text-to-image generator. This fork supports:
+# **Stable Diffusion Dream Script**
 
-1. An interactive command-line interface that accepts the same prompt
-   and switches as the Discord bot.
+This is a fork of [CompVis/stable-diffusion](https://github.com/CompVis/stable-diffusion), the open source text-to-image generator. It provides a streamlined process with various new features and options to aid the image generation process.
 
-2. A basic Web interface that allows you to run a local web server for
-   generating images in your browser.
+_Note: This fork is rapidly evolving. Please use the [Issues](https://github.com/lstein/stable-diffusion/issues) tab to report bugs and make feature requests._
```

Comment:
"AND PLEASE ACTUALLY FILL OUT THE MOTHER-@(&#^$ TEMPLATE"

But nicer than that, of course 😂

Reply:
Added this: "Be sure to use the provided templates. They will help aid diagnose issues faster."